### DISCUSSION TITLE
Programa de la materia

### DISCUSSION THREAD

**Discussion by Clirussa** [INGCOM-UNRN-P1/cursada-2023#201] [Preguntas y respuestas] [answered]
Hola, escribo porque no encuentro el programa de la materia en el campus. Lo podrían compartir por acá por favor. Gracias.

**Comment by Nikzza** [comment]
https://campusbimodal.unrn.edu.ar/mod/page/view.php?id=721303

**Reply by Clirussa** [reply]
Pero ese es el cronograma, no el programa. Gracias igual.

**Comment by martinvilu** [comment, marked as answer]
No está completo, además de lo que esta no pasó por el circuito de revisión.

En esta semana si terminamos de revisarlo nosotros, te mando el preliminar.

**Comment by martinvilu** [comment]
La versión preliminar, sujeta a cambios, aunque no debieran de ser demasiado grandes. (falta que complete el circuito de revisión con las autoridades)

[CL 2023-B6003-Programación I-ICOMP 2021-Ingeniería en Computación-Version aceptada.pdf](https://github.com/INGCOM-UNRN-P1/cursada-2023/files/13194176/CL.2023-B6003-Programacion.I-ICOMP.2021-Ingenieria.en.Computacion-Version.aceptada.pdf)

**Reply by Clirussa** [reply]
Muchas gracias
